ansible/health_checker.py

```diff
@@ -0,0 +1,98 @@
+import imp
+import os
+import logging
+import sys
+import json
+import argparse
+
+
+ANSIBLE_DIR = os.path.abspath(os.path.dirname(__file__))
+SONIC_MGMT_DIR = os.path.dirname(ANSIBLE_DIR)
+TESTBED_FILE = 'testbed.yaml'
+SONIC_TESTBED_HEALTH_FILE = 'testbed_ping.json'
+
+root = logging.getLogger()
+root.setLevel(logging.DEBUG)
+
+handler = logging.StreamHandler(sys.stdout)
+handler.setLevel(logging.DEBUG)
+formatter = logging.Formatter('%(levelname)s - %(message)s')
+handler.setFormatter(formatter)
+root.addHandler(handler)
+
+def get_dut_tbname_map():
+    """Return a dictionary containing mapping from dut hostname to testbed name."""
+    testbed = imp.load_source('testbed', os.path.join(SONIC_MGMT_DIR, 'tests/common/testbed.py'))
+    dut_tbname_map = {}
+    for tbname, tb in testbed.TestbedInfo(TESTBED_FILE).testbed_topo.items():
+        for duthost in tb['duts']:
+            dut_tbname_map[duthost] = tbname
+
+    return dut_tbname_map
+
+def parse_ping_result(ping_results, dut_tbname_map, skip_testbeds):
+    """Parse ping results of devutils to kusto table data format."""
+    final_results = []
+    for dut_ping_result in ping_results:
+        dut_result = {}
+        dut_result['DeviceName'] = dut_ping_result['Host']
+        dut_result['IP'] = dut_ping_result['Hostname']
+        dut_result['IcmpReachability'] = 1 if dut_ping_result['Ping result'] == 'Success' else 0
+        dut_result['ConsoleReachability'] = 0
+        dut_result['SshReachability'] = 0
```

Comment:
If you use bool to indicate success or fail, will that mislead user to think console and ssh are always reachable? Or just keep icmp reachability?

Reply:
@xwjiang-ms ssh and console are not implemented yet. Just have these column, not used them yet, so set them to 0.
0 means false.

Reply:
Set it to null for ssh and console.